### DISCUSSION TITLE
v2.3.1

### DISCUSSION THREAD

**Discussion by UnchartedBull** [UnchartedBull/OctoDash#2725] [General]
This is just a small release to make the notifications less obtrusive. Included stuff:

- new icons for printer status
- printer notifications are now shown in the bottom right, instead of being shown as a notification
- increased hitbox for notification center triangles

[Full Changelog](https://github.com/UnchartedBull/OctoDash/compare/v2.3.0...v2.3.1)

**Fixed bugs:**

- Blue fan icon is actually roll of filament  [\#2711](https://github.com/UnchartedBull/OctoDash/issues/2711)
- Control Panel "triangle" too small to be touched by finger [\#2707](https://github.com/UnchartedBull/OctoDash/issues/2707)
- Printer information impairs touch responsiveness [\#2720](https://github.com/UnchartedBull/OctoDash/issues/2720)

**Closed issues:**

- 0000 delete [\#2713](https://github.com/UnchartedBull/OctoDash/issues/2713)
- Can't load Electron library when accessing dev environment throught localhost in Chrome [\#2712](https://github.com/UnchartedBull/OctoDash/issues/2712)
- HyperPixel 4.0 on RPi 4 - How to: [\#2309](https://github.com/UnchartedBull/OctoDash/issues/2309)

**Merged pull requests:**

- Increase notification center icon hitbox [\#2724](https://github.com/UnchartedBull/OctoDash/pull/2724) ([UnchartedBull](https://github.com/UnchartedBull))
- update status icons [\#2723](https://github.com/UnchartedBull/OctoDash/pull/2723) ([UnchartedBull](https://github.com/UnchartedBull))
- fix annoying notifications [\#2722](https://github.com/UnchartedBull/OctoDash/pull/2722) ([UnchartedBull](https://github.com/UnchartedBull))
- Bump ts-node from 10.5.0 to 10.6.0 [\#2721](https://github.com/UnchartedBull/OctoDash/pull/2721) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @typescript-eslint/eslint-plugin from 5.12.1 to 5.13.0 [\#2719](https://github.com/UnchartedBull/OctoDash/pull/2719) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @typescript-eslint/parser from 5.12.1 to 5.13.0 [\#2718](https://github.com/UnchartedBull/OctoDash/pull/2718) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump eslint from 8.9.0 to 8.10.0 [\#2716](https://github.com/UnchartedBull/OctoDash/pull/2716) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump xliff from 6.0.1 to 6.0.3 [\#2715](https://github.com/UnchartedBull/OctoDash/pull/2715) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump electron from 17.0.1 to 17.1.0 [\#2706](https://github.com/UnchartedBull/OctoDash/pull/2706) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @angular/cli from 13.2.4 to 13.2.5 [\#2705](https://github.com/UnchartedBull/OctoDash/pull/2705) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @angular/platform-browser from 13.2.3 to 13.2.4 [\#2704](https://github.com/UnchartedBull/OctoDash/pull/2704) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @angular/forms from 13.2.3 to 13.2.4 [\#2703](https://github.com/UnchartedBull/OctoDash/pull/2703) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @angular/router from 13.2.3 to 13.2.4 [\#2702](https://github.com/UnchartedBull/OctoDash/pull/2702) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @angular/common from 13.2.3 to 13.2.4 [\#2701](https://github.com/UnchartedBull/OctoDash/pull/2701) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @angular/language-service from 13.2.3 to 13.2.4 [\#2700](https://github.com/UnchartedBull/OctoDash/pull/2700) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @types/node from 17.0.19 to 17.0.21 [\#2699](https://github.com/UnchartedBull/OctoDash/pull/2699) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @angular-devkit/build-angular from 13.2.4 to 13.2.5 [\#2698](https://github.com/UnchartedBull/OctoDash/pull/2698) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @angular/animations from 13.2.3 to 13.2.4 [\#2697](https://github.com/UnchartedBull/OctoDash/pull/2697) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @angular/platform-browser-dynamic from 13.2.3 to 13.2.4 [\#2696](https://github.com/UnchartedBull/OctoDash/pull/2696) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @angular/material from 13.2.3 to 13.2.4 [\#2695](https://github.com/UnchartedBull/OctoDash/pull/2695) ([dependabot[bot]](https://github.com/apps/dependabot))
- Bump @typescript-eslint/eslint-plugin from 5.12.0 to 5.12.1 [\#2689](https://github.com/UnchartedBull/OctoDash/pull/2689) ([dependabot[bot]](https://github.com/apps/dependabot))

<hr /><em>This discussion was created from the release <a href='https://github.com/UnchartedBull/OctoDash/releases/tag/v2.3.1'>v2.3.1</a>.</em>

**Comment by PapsDizzy** [comment]
Can not download and install ! the update to 2.3.0 was all good.
![IMG_20220302_221006.jpg](https://user-images.githubusercontent.com/83864347/156450907-8e2d0355-a70c-4636-bbb4-92b28c74f35e.jpg)

**Reply by PapsDizzy** [reply]
now i have done the update via putty

**Reply by UnchartedBull** [reply]
oh damn, that's probably a bug in 2.3.0 (which is in 2.3.1 now as well). Let me check whether that's an easy fix

**Reply by PapsDizzy** [reply]
no problem i have see it and post it for you as information.
via putty my update goes well. now is all fine, no spamming notifications  popups.
the notifications are well placed in the lower right. the new icons are also cool.

**Reply by UnchartedBull** [reply]
Great that you like it! I've updated the 2.3.1 release to fix the download functionality. If you want those included just execute the update script again. But there is nothing else in there, so if you're fine with executing the update script once v3 is out you won't miss out on anything :)

**Reply by javierzg** [reply]
Hello.
A big greeting.
I have the same problem, do you have a correction or should it be downloaded from putty, what would be the steps via putty?

**Reply by jneilliii** [reply]
SSH to the pi and run the commands listed in manual update steps on the wiki.

https://github.com/UnchartedBull/OctoDash/wiki/Update#manual-update

**Reply by javierzg** [reply]
Thank you very much jneillii
updated from putty.
Thanks
UnchartedBull, for this great work, everything is fine working with OctoDash

**Comment by borisott** [comment]
![Bildschirmfoto 2022-03-16 um 15 27 36](https://user-images.githubusercontent.com/75563318/158613590-a0b87ffe-3018-4fba-b82c-b2fd3a9bc162.jpg)
Hello,
when I try to update to 2.3.1 I get the known javascript error.
Trying the script I get the shown error.

Any suggestions?

Thx

Boris

**Reply by UnchartedBull** [reply]
https://github.com/UnchartedBull/OctoDash/issues/2727
